### PR TITLE
fix(SecretListPage): add workspace breadcrumbs on secrets list page

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1286

## Description

Add workspace switcher on the secret list page

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/pages/SecretsListPage.tsx [hunk 1]
```diff
@@ -32,6 +33,13 @@ const SecretsListPage: React.FC<React.PropsWithChildren<unknown>> = () => {
               </HelpTopicLink>
             </>
           }
+          breadcrumbs={[
+            ...breadcrumbs,
+            {
+              path: '#',
+              name: 'Releases',
```
Comment (abhinandan13jan) [inline src/pages/SecretsListPage.tsx:40]: This should be Secrets

**Review by rohitkrai03 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**